mesonbuild/compilers/fortran.py

```diff
@@ -101,10 +101,11 @@ def module_name_to_filename(self, module_name: str) -> str:
 
         return filename
 
-    def find_library(self, libname: str, env: 'Environment', extra_dirs: T.List[str],
-                     libtype: LibType = LibType.PREFER_SHARED, lib_prefix_warning: bool = True) -> T.Optional[T.List[str]]:
+    def find_library2(self, libname: str, env: 'Environment', extra_dirs: T.List[str],
```

Comment:
Perhaps a name like find_library_from_cache would be more understandable?

Reply:
done